packages/transformer/src/IModelTransformer.ts

```diff
@@ -1526,24 +1526,24 @@ export class IModelTransformer extends IModelExportHandler {
       sourceElement,
       { binaryGeometry: this._options.cloneUsingBinaryGeometry }
     );
+
+    const targetElementId = this.context.findTargetElementId(sourceElement.id);
```

Comment:
I also wonder, should all of this logic live in onExportElement instead?
Right now we probably only get the targetElementId if someone explicitly remapped it (I think) but its possible that rootSubject could be remapped to a nonRootSubject by virtue of having the same fedguid due to some prior explicit remapping.